### DISCUSSION TITLE
Sri Lanka - how long to wait? :(

### DISCUSSION THREAD

**Discussion by Harindulk** [community/community#8563] [Other Feature Feedback, Questions, & Ideas]
how much longer will it take to give it to Sri Lanka, I have been on the waiting list for a long time 🥲🥲 :(

**Comment by MrKrishnaAgarwal** [comment]
![image](https://user-images.githubusercontent.com/100597998/173173888-03959b8a-b9f7-434e-b8f2-f23a80bf7e32.png)
